src/effects/native/gatereffect.cpp

```diff
@@ -0,0 +1,236 @@
+#include "effects/native/gatereffect.h"
+
+namespace {
+    // Gain when the gate is open, higher than 1 to still have a decent level
```

Comment:
We omit indentation within (possibly nested) namespaces to keep the code readable.